### PR TITLE
connect single contract no moralis

### LINKED ISSUE #17: Connect `Single Contract` -> `getJob`
- Use Ethers to pull up the job ID referenced in the URL from our contract (ie. `engineerdao.com/contract/1` is `1`)
- Pull up logs for that contract, where we will find the contract information
- Populate fields in Single Contract with information from the contract
- Display a loader or skeleton loader until all contract fields are resolved

### PR DESCRIPTION
**Pull request by deweller**
Shows a single contract data on the frontend.

Includes latest contract build

Closes #17

### CODE REVIEW

File: frontend/src/components/data-table/DataTable.tsx [hunk 1]
```diff
@@ -1,46 +1,76 @@
...
+  filename: string;
+  link: string;
+}
+
+// interface ICurrency {
+//   crypto_value: number;
+//   crypto_suffix: string;
+//   fiat_value?: number;
+//   fiat_suffix?: string;
+// }
```
Comment (reisr3) [inline frontend/src/components/data-table/DataTable.tsx:22]: I'd just delete this if we don't need it.

File: frontend/src/components/single-contract/StatusChip.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import classNames from 'classnames';
 import React from 'react';
-import { ContractStatus } from '../../enums/ContractStatus';
+import { IJobState } from 'interfaces/IJobData';
```
Comment (reisr3) [inline frontend/src/components/single-contract/StatusChip.tsx:3]: Isn't this an enum? I'd move it into its own file, kinda confusing here.

File: frontend/src/interfaces/IJobData.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+// import { ContractStatus } from '../enums/ContractStatus';
+import { BigNumber } from 'ethers';
+
+export enum IJobState {
```
Comment (reisr3) [inline frontend/src/interfaces/IJobData.ts:4]: I would just break this file up a bit more.

Comment (deweller) [reply frontend/src/interfaces/IJobData.ts:4]: Made #67 to remind us to do this.

**Pull request merged**